nasa-gibs/worldview
# WV-2996 Implement Notification Wireframes

## Description

Streamline DDV notification messages

## How To Test

- `git checkout wv-2996-notification-wireframe`
- `npm run watch`
- Open this [link](http://localhost:3000/?v=-139.80289154597668,-18.48464428620524,10.697155836633641,65.03939821375037&l=Reference_Labels_15m(hidden),Reference_Features_15m(hidden),Coastlines_15m,HLS_Shortwave_Infrared_Landsat(bandCombo=%7B%22r%22%3A%22B07%22;%22g%22%3A%22B05%22;%22b%22%3A%22B04%22%7D),HLS_False_Color_Urban_Landsat(bandCombo=%7B%22r%22%3A%22B07%22;%22g%22%3A%22B06%22;%22b%22%3A%22B04%22%7D),HLS_False_Color_Vegetation_Landsat(bandCombo=%7B%22r%22%3A%22B06%22;%22g%22%3A%22B05%22;%22b%22%3A%22B04%22%7D),HLS_False_Color_Landsat(bandCombo=%7B%22r%22%3A%22B05%22;%22g%22%3A%22B04%22;%22b%22%3A%22B03%22%7D),HLS_Customizable_Landsat(bandCombo=%7B%22r%22%3A%22B07%22;%22g%22%3A%22B05%22;%22b%22%3A%22B04%22%7D),VIIRS_NOAA20_CorrectedReflectance_TrueColor(hidden),VIIRS_SNPP_CorrectedReflectance_TrueColor(hidden),MODIS_Aqua_CorrectedReflectance_TrueColor(hidden),MODIS_Terra_CorrectedReflectance_TrueColor&lg=true&t=2024-02-11-T02%3A00%3A00Z)
- Verify that it says "Multiple Layer Alerts" in the top right corner
- Expand the dropdown to confirm that it contains the layer alerts
- Remove all but one of the layers with alerts to verify that the sole remaining alert is displayed without the dropdown.
